templates/default/event/eventlistfilter.html

```diff
@@ -30,12 +30,19 @@
 							<input type="hidden" name="userand" value="0">
 							<input type="checkbox" name="userand" value="1" onchange="document.filter.submit();"{if isset($filter.userand) && $filter.userand} checked{/if}>
 						</label>
-						<SELECT size="1" name="a[]" OnChange="document.filter.submit();" class="lms-ui-multiselect lms-ui-multiselect-filter" multiple data-default-value="{trans("- all -")}">
-							<OPTION value="-1" {if is_array($filter.userid) && in_array('-1', $filter.userid)} selected{/if} data-exclusive>- {trans("unassigned")} -</OPTION>
-							{foreach $userlist as $user}
-								<OPTION value="{$user.id}"{if is_array($filter.userid) && in_array($user.id, $filter.userid)} SELECTED{/if}{if !$user.access} disabled{/if}>{$user.rname|truncate:40:"...":true} ({$user.login})</OPTION>
-							{/foreach}
-						</SELECT>
+                        <SELECT size="1" name="a[]" OnChange="document.filter.submit();"
+                                class="lms-ui-multiselect lms-ui-multiselect-filter" multiple
+                                data-default-value="{trans("- all -")}">
+                            <OPTION value="-1" {if is_array($filter.userid) && in_array('-1', $filter.userid)} selected{/if}
+                                    data-exclusive>{trans("unassigned")}
+                            </OPTION>
+                            {foreach $userlist as $user}
+                                <OPTION $filter.userid)}&& SELECTED{ in_array($user.id, is_array($filter.userid)
```

Comment:
a tu nie ma czasem jakiegoś błędu składniowego smarty? dziwnie to wygląda...

Reply:
Tak mi właśnie poporawiał phpstorm przy użyciu innej templatki... niby lepiej a jednak nie idealnie.

Reply:
Pytam po raz 3-ci: masz ustawione dla plików *.html traktowanie ich w PHPStorm jako szablonów Smarty?

Reply:
Ustawiłem to jedynie per ten jeden plik ponieważ globalnie nie mogłem tego znaleźć.

Reply:
To co zostawiamy z błędem składniowym?

Reply:
Jest już nawet dokumentacja:
https://github.com/chilek/lms-plus/wiki/Zmienne-konfiguracyjne-LMS-Plus-phpui#phpuitimetable_hide_disabled_users
;-)

Reply:
https://github.com/lmsgit/lms/pull/1720
Kto by pomyślał że ta opcja narobi błędów...

Swoją drogą travis może je również wychwytywać?

Reply:
Zobacz czy Tobie też robi dokładnie taki sam code reformat.